>>> 'Pkg.add("SymPy")' log
INFO: Cloning cache of SymPy from git://github.com/JuliaPy/SymPy.jl.git
INFO: Installing BinDeps v0.4.5
INFO: Installing Conda v0.5.1
INFO: Installing MacroTools v0.3.5
INFO: Installing PyCall v1.10.0
INFO: Installing RecipesBase v0.0.6
INFO: Installing SHA v0.3.1
INFO: Installing SymPy v0.4.0
INFO: Installing URIParser v0.1.8
INFO: Building Conda
INFO: Building PyCall
INFO: Using the Python distribution in the Conda package by default.
To use a different Python version, set ENV["PYTHON"]="pythoncommand" and re-run Pkg.build("PyCall").
Fetching package metadata ....
WARNING: The remote server could not find the noarch directory for the
requested channel with url: https://conda.anaconda.org/poliastro

It is possible you have given conda an invalid channel. Please double-check
your conda configuration using `conda config --show`.

If the requested url is in fact a valid conda channel, please request that the
channel administrator create `noarch/repodata.json` and associated
`noarch/repodata.json.bz2` files, even if `noarch/repodata.json` is empty.
$ mkdir noarch
$ echo '{}' > noarch/repodata.json
$ bzip2 -k noarch/repodata.json
.......
Solving package specifications: .

# All requested packages already installed.
# packages in environment at /home/vagrant/.julia/v0.4/Conda/deps/usr:
#
numpy                     1.12.0                   py27_0  
INFO: PyCall is using /home/vagrant/.julia/v0.4/Conda/deps/usr/bin/python (Python 2.7.12) at /home/vagrant/.julia/v0.4/Conda/deps/usr/bin/python, libpython = /home/vagrant/.julia/v0.4/Conda/deps/usr/lib/libpython2.7
INFO: /home/vagrant/.julia/v0.4/PyCall/deps/deps.jl has not changed
INFO: /home/vagrant/.julia/v0.4/PyCall/deps/PYTHON has not changed
INFO: Package database updated
INFO: METADATA is out-of-date â€” you may not have the latest version of SymPy
INFO: Use `Pkg.update()` to get the latest versions of your packages

>>> 'Pkg.test("SymPy")' log
Julia Version 0.4.7
Commit ae26b25 (2016-09-18 16:17 UTC)
Platform Info:
  System: Linux (x86_64-pc-linux-gnu)
  CPU: Intel(R) Xeon(R) CPU E3-1241 v3 @ 3.50GHz
  WORD_SIZE: 64
           Ubuntu 14.04.5 LTS
  uname: Linux 3.13.0-108-generic #155-Ubuntu SMP Wed Jan 11 16:58:52 UTC 2017 x86_64 x86_64
Memory: 2.9392738342285156 GB (975.2734375 MB free)
Uptime: 39390.0 sec
Load Avg:  0.8525390625  0.9697265625  0.986328125
Intel(R) Xeon(R) CPU E3-1241 v3 @ 3.50GHz: 
       speed         user         nice          sys         idle          irq
#1  3500 MHz    1900257 s       7210 s     155950 s    1451881 s        274 s
#2  3500 MHz    1260987 s          6 s     139414 s    2315450 s         14 s

  BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY Nehalem)
  LAPACK: libopenblas64_
  LIBM: libopenlibm
  LLVM: libLLVM-3.3
Environment:
  TERM = vt100
  LD_LIBRARY_PATH = :/usr/local/lib/
  PATH = /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/home/vagrant/julia/bin/
  JAVA_HOME = /usr/lib/jvm/java-7-oracle
  HOME = /home/vagrant

Package Directory: /home/vagrant/.julia/v0.4
2 required packages:
 - JSON                          0.8.3
 - SymPy                         0.4.0
8 additional packages:
 - BinDeps                       0.4.5
 - Compat                        0.18.0
 - Conda                         0.5.1
 - MacroTools                    0.3.5
 - PyCall                        1.10.0
 - RecipesBase                   0.0.6
 - SHA                           0.3.1
 - URIParser                     0.1.8
INFO: Testing SymPy
WARNING: pyeval is deprecated.  Use py"str(x)" instead.  Use $ interpolation to substitute Julia variables and expressions into Python.
 in depwarn at deprecated.jl:73
 in pyeval at /home/vagrant/.julia/v0.4/PyCall/src/pyeval.jl:58
 in pyeval at /home/vagrant/.julia/v0.4/PyCall/src/pyeval.jl:43
 in print at strings/io.jl:8
 in print_to_string at strings/io.jl:36
 in map at abstractarray.jl:1305
 in lambdify at /home/vagrant/.julia/v0.4/SymPy/src/lambdify.jl:148
 in lambdify at /home/vagrant/.julia/v0.4/SymPy/src/lambdify.jl:142
 in convert at /home/vagrant/.julia/v0.4/SymPy/src/types.jl:168
 in include at ./boot.jl:261
 in include_from_node1 at ./loading.jl:333
 in include at ./boot.jl:261
 in include_from_node1 at ./loading.jl:333
 in process_options at ./client.jl:280
 in _start at ./client.jl:378
while loading /home/vagrant/.julia/v0.4/SymPy/test/tests.jl, in expression starting on line 38

signal (11): Segmentation fault
unknown function (ip: 0x7f600eb4f4cf)
unknown function (ip: 0x7f600eaf6230)
unknown function (ip: 0x7f600eb16094)
unknown function (ip: 0x7f600eb1682d)
unknown function (ip: 0x7f600ea91589)
unknown function (ip: 0x7f600ea88cce)
unknown function (ip: 0x7f600e23319c)
unknown function (ip: 0x7f600e23444a)
unknown function (ip: 0x7f600e23462c)
jl_trampoline at /home/vagrant/julia/bin/../lib/julia/libjulia.so (unknown line)
jl_apply_generic at /home/vagrant/julia/bin/../lib/julia/libjulia.so (unknown line)
anonymous at test.jl:90
do_test at test.jl:50
anonymous at /home/vagrant/.julia/v0.4/SymPy/test/tests.jl:105
unknown function (ip: 0x7f600e276b13)
unknown function (ip: 0x7f600e27772c)
jl_load at /home/vagrant/julia/bin/../lib/julia/libjulia.so (unknown line)
include at ./boot.jl:261
jl_apply_generic at /home/vagrant/julia/bin/../lib/julia/libjulia.so (unknown line)
include_from_node1 at ./loading.jl:333
jl_apply_generic at /home/vagrant/julia/bin/../lib/julia/libjulia.so (unknown line)
unknown function (ip: 0x7f600e262266)
unknown function (ip: 0x7f600e2615cf)
unknown function (ip: 0x7f600e276a5c)
unknown function (ip: 0x7f600e27772c)
jl_load at /home/vagrant/julia/bin/../lib/julia/libjulia.so (unknown line)
include at ./boot.jl:261
jl_apply_generic at /home/vagrant/julia/bin/../lib/julia/libjulia.so (unknown line)
include_from_node1 at ./loading.jl:333
jl_apply_generic at /home/vagrant/julia/bin/../lib/julia/libjulia.so (unknown line)
process_options at ./client.jl:284
_start at ./client.jl:378
unknown function (ip: 0x7f600b483e79)
jl_apply_generic at /home/vagrant/julia/bin/../lib/julia/libjulia.so (unknown line)
unknown function (ip: 0x401c87)
unknown function (ip: 0x40186f)
__libc_start_main at /lib/x86_64-linux-gnu/libc.so.6 (unknown line)
unknown function (ip: 0x4018b5)
================================[ ERROR: SymPy ]================================

failed process: Process(`/home/vagrant/julia/bin/julia --check-bounds=yes --code-coverage=none --color=no /home/vagrant/.julia/v0.4/SymPy/test/runtests.jl`, ProcessSignaled(11)) [0]

================================================================================
ERROR: SymPy had test errors
 in error at ./error.jl:21
 in test at pkg/entry.jl:803
 in anonymous at pkg/dir.jl:31
 in cd at file.jl:22
 in cd at pkg/dir.jl:31
 in test at pkg.jl:71
 in process_options at ./client.jl:257
 in _start at ./client.jl:378

>>> End of log
